app.js

```diff
@@ -6,18 +6,35 @@ const app = express();
 const PORT = 3001;
 
 mongoose.connect('mongodb://localhost:27017/coderingevent1', {
-  useNewUrlParser: true,
-  useUnifiedTopology: true
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    useFindAndModify: false
 });
 
 app.get('/', (req, res) => {
-  res.send(200);
+    res.send(200);
 });
 
-app.get("/users", async (req, res) => {
-  let users = await User.find({});
+app.get("/users/", async (req, res) => {
```

Comment:
Please fix this. Do not change things that don't need to be changed. Remove the /